kore/test/Test/Kore/Step/Simplification/Condition.hs

```diff
@@ -76,13 +76,18 @@ test_simplify_local_functions =
     , test "contradiction: \"one\" = f(x) ∧ f(x) = \"two\"" fString (Left  strOne) (Right strTwo)
     , test "contradiction: \"one\" = f(x) ∧ \"two\" = f(x)" fString (Left  strOne) (Left  strTwo)
     , test "contradiction: f(x) = \"one\" ∧ \"two\" = f(x)" fString (Right strOne) (Left  strTwo)
+    -- Ignore Defined marker
+    , testDefined "contradiction: f(g(x)) = a ∧ f(g(x)) = b" (Right fg) (Right a) (Right b)
+    , testDefined "contradiction: f(g(x)) = a ∧ f(g(x)) = b" (Left fg) (Right a) (Right b)
```

Comment:
What is the difference between `Right fg` and `Left fg`?

Reply:
`Right fg` will apply `mkDefined` to the `f(g(x)` in the second equals, and `Left fg` to the one in the first.